numba/cuda/compiler.py

```diff
@@ -677,27 +568,27 @@ def regs_per_thread(self):
         '''
         The number of registers used by each thread for this kernel.
         '''
-        return self._func.get().attrs.regs
+        return self._codelibrary.get_cufunc().attrs.regs
 
     def inspect_llvm(self):
         '''
         Returns the LLVM IR for this kernel.
         '''
-        return str(self._func.ptx.llvmir)
+        return self._codelibrary.get_asm_str()
```

Comment:
Is this right? Should it be `get_llvm_str`?

Reply:
It's wrong - it should be as you suggest - now fixed.